### ISSUE TITLE
Task 33 (mobile footer)

### ISSUE THREAD

**Issue by pmvald** [Appers-lab/sss-theme#51]
Prerequisite: 
- [design guide](http://appers-lab.com/guides/design-system) 
- [mobile design guide](http://appers-lab.com/guides/mobile-design-system) 

MobileKit first page: scroll to the bottom -> we want to design a footer similar to this. You can grab the footer materials from [this demo page] (http://billihomes.ca:4000/about-us/). Make a class called `s-footer` in styles/_footers.m.scss. Details up to you

**Comment by miyabitanimchi** [comment]
Hello @pmvald, I would like to take this task.

**Comment by pmvald** [comment]
@miyabitanimchi sure!

**Comment by miyabitanimchi** [comment]
<img width="765" alt="スクリーンショット 2021-06-30 午前2 50 38" src="https://user-images.githubusercontent.com/69213541/123941740-506e9400-d94f-11eb-9704-829da368dbdd.png">
Hi @pmvald , I tried to push to my branch miyabi but then this error occurred. I researched about this error and I created my personal access token then, but I could not figure out where I could set the token to... It says it's "temporarily disabled as part of a brownout" so I should just wait? </br>
I'm sorry it's not about this task, but I would really appreciate if I could have some solutions from you if you know something. Thank you.
